### PR TITLE
add initial table creates to sql init

### PR DESCRIPTION
**Pull request by morganjcw**

### CODE REVIEW

**Comment by morganjcw** [comment]
Need a maintainer to approve running workflows but @strikefvela @helloimalastair tagging you here

File: backend/db_init.sql
Comment (skrtu) [inline backend/db_init.sql:1]: can we remove the user table? we should just use the `user_id` field from Auth0 to reference other objects I think.

File: backend/db_init.sql
Comment (skrtu) [inline backend/db_init.sql:12]: can we add a `user_id` here?

**Review by skrtu — CHANGES_REQUESTED** [review]
Hey if you could review my comments before we merge.

**Review by skrtu — APPROVED** [review]
LGTM 👍

**Comment by morganjcw** [comment]
@skrtu can I get another review on this?

**Pull request merged**
